packages/flutter/lib/src/foundation/change_notifier.dart

```diff
@@ -109,11 +102,15 @@ class _ListenerEntry extends LinkedListEntry<_ListenerEntry> {
 ///
 ///  * [ValueNotifier], which is a [ChangeNotifier] that wraps a single value.
 class ChangeNotifier implements Listenable {
-  LinkedList<_ListenerEntry>? _listeners = LinkedList<_ListenerEntry>();
+  int _length = 0;
```

Comment:
_count might be a better name for this, since it's not measuring the actual allocated length of anything but the count of active listeners within the _listeners list.